packages/@aws-cdk/aws-servicediscovery/lib/http-namespace.ts

```diff
@@ -0,0 +1,47 @@
+import cdk = require('@aws-cdk/cdk');
+import { CfnHttpNamespace } from './servicediscovery.generated';
+
+export interface HttpNamespaceProps {
+  /**
+   * A name for the HttpNamespace.
+   *
+   * @default CloudFormation-generated name
```

Comment:
If this actually has a default the declaration for name should be `name?`

Reply:
oh, no. this should be required.